tests/test_block_send_session.rs

```diff
@@ -0,0 +1,159 @@
+use bytes::Bytes;
```

Comment:
this test looks like not related to discovery interval issue, could you please move it to other PR?

Reply:
OK, I'll issue a PR fix those tests first